caffe2: UpsampleBilinear support for scales

d4l3k opened the pull request:
Summary: This updates UpsampleBilinearOp and UpsampleBilinearGradientOp to support scales to bring it inline with ResizeNearestOp https://github.com/pytorch/pytorch/pull/12720.

Differential Revision: D10416228

houseroad commented on caffe2/python/operator_test/upsample_op_test.py:
```diff
@@ -40,52 +40,72 @@ def test_upsample(self, height_scale, width_scale, height, width,
+
+        ops = [
+            (
+                core.CreateOperator(
```
Lint the code? the format looks a little bit weird here.

d4l3k replied:
Linted

d4l3k committed:
caffe2: UpsampleBilinear support for scales (#12736)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/12736

This updates UpsampleBilinearOp and UpsampleBilinearGradientOp to support scales to bring it inline with ResizeNearestOp https://github.com/pytorch/pytorch/pull/12720.

Reviewed By: houseroad

Differential Revision: D10416228

fbshipit-source-id: f687a16b238e81425a27a8d7037c4c3e602ed958
